code/modules/antagonists/heretic/items/heretic_necks.dm

```diff
@@ -50,3 +50,41 @@
 	icon = 'icons/obj/antags/eldritch.dmi'
 	icon_state = "eye_medalion"
 	w_class = WEIGHT_CLASS_SMALL
+
+
+// The amulette conversion tool used by moon heretics
+/obj/item/clothing/neck/heretic_focus/moon_amulette
+	name = "Moonlight Amulette"
+	desc = "A piece of the mind, the sight and the moon. Gazing into it makes your head spin and hear whispers of laughter and joy."
+	icon = 'icons/obj/antags/eldritch.dmi'
+	icon_state = "moon_amulette"
+	w_class = WEIGHT_CLASS_SMALL
+	// How much damage does this item do to the targets sanity?
+	var/sanity_damage = 20
+
+/obj/item/clothing/neck/heretic_focus/moon_amulette/attack(mob/living/target, mob/living/user, params)
+	var/mob/living/carbon/human/hit = target
+	if(IS_HERETIC(user))
+		if(hit.can_block_magic())
+			return
+		if(hit.mob_mood)
+			switch(hit.mob_mood.sanity_level)
+				if(SANITY_LEVEL_GREAT, SANITY_LEVEL_NEUTRAL, SANITY_LEVEL_DISTURBED)
+					user.balloon_alert(user, "their mind is too strong!")
+					hit.add_mood_event("Moon Amulette Insanity", /datum/mood_event/amulette_insanity)
+					hit.mob_mood.set_sanity(hit.mob_mood.sanity - sanity_damage)
+				if(SANITY_LEVEL_CRAZY, SANITY_LEVEL_INSANE)
+					user.balloon_alert(user, "their mind bends to see the truth!")
+					hit.apply_status_effect(/datum/status_effect/moon_converted)
+					user.log_message("made [target] insane.", LOG_GAME)
+					hit.log_message("was driven insane by [user]")
+	else
+		user.balloon_alert(user, "you feel a presence watching you")
+		user.add_mood_event("Moon Amulette Insanity", /datum/mood_event/amulette_insanity)
+		user.mob_mood.set_sanity(user.mob_mood.sanity - 50)
+	. = ..()
```

Comment:
early return on !isheretic and !hit.mobmood

Reply:
done